packages/backend/src/peer/errors.ts

```diff
@@ -24,6 +26,7 @@ export const errorToMessage: {
   [key in PeerError]: string
 } = {
   [PeerError.DuplicateIncomingToken]: 'duplicate incoming token',
+  [PeerError.DuplicatePeer]: 'duplicate peer',
```

Comment:
Would it be better to provide a more specific error message rather than simply "Duplicate peer"?

Reply:
Updated to be more specific